projects.json

```diff
@@ -4,7 +4,7 @@
       "name": "CO2-Labels for University Canteen",
       "image": "/data_temp/images/canteen.png",
       "organisation_name": "sneep Erlangen e.V.",
-      "organisation_image": "/data_temp/images/sneep.png",
+      "organization_image": "/data_temp/images/sneep.png",
```

Comment:
IMO, we should undo this change, as it is (1) inconsistent with `organisation_name` above (2) unrelated to the overall goal of this PR. Updating this probably also means we'll update `organization_image` elsewhere in this PR.

Relatedly: as we work on this project, I think we'll need to decide whether we internally use American English or British English for various spellings. I don't think it really matters as long as we pick one, and that decision is much less important than what we display _publicly_ (which should, ideally, be internationalized).